packages/graphql/src/schemaTypes/objectTypes/gql-WizardErroredFramework.ts

```diff
@@ -0,0 +1,20 @@
+import { objectType } from 'nexus'
+
+export const WizardErroredFramework = objectType({
+  name: 'WizardErroredFramework',
+  description: '',
+  node: 'path',
+  definition (t) {
+    t.string('name', {
+      description: 'The display name of the framework',
+    }),
+
+    t.string('path', {
+      description: 'The location of the framework\'s package.json file',
+    }),
+
+    t.string('reason', {
```

Comment:
@marktnoonan do we need the `reason` or `name` fields at all? It wasn't clear to me how we were planning to use them

Reply:
I removed them for now, let's get this over the line